### PR TITLE
Adding string_practice to repo to be graded.

### PR DESCRIPTION
**Pull request by anthonyfalzetti**
[#88505492]

### CODE REVIEW

File: methods_and_flow_control/string_practice.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+practice_string = 'this is a string to practice with'
+
+puts practice_string
+puts practice_string.capitalize
+puts practice_string.upcase
+puts practice_string.gsub('string', "'string'").capitalize
+puts "The string '#{practice_string}' has #{practice_string.length} characters"
+puts practice_string.reverse
+puts "#{practice_string.replace('practice practice practice')}!"
```
Comment (jasonnoble) [inline methods_and_flow_control/string_practice.rb]: Can you find a string method to pull the word practice out of `practice_string`?  You're just replacing the entire string with `practice practice practice`, which is pretty close to `puts "practice practice practice!".

**Comment by jasonnoble** [comment]
Needs rebasing...

**Comment by jasonnoble** [comment]
Please add the story # to your commit message

**Comment by wb-davinci-ror-course-projects** [comment]
[#88505492]

**Pull request merged**